Backend/routes/userRoutes.js

```
@@ -1,103 +1,116 @@
-    const users = await User.find({}, "uid fullName email role");
-    res.status(200).json(users);
-  } catch (error) {
-    console.error("❌ Error fetching users:", error);
-    res.status(500).json({ message: "Internal server error" });
-  }
+    try {
+        const users = await User.find({}, "uid fullName email role");
+        res.status(200).json(users);
+    } catch (error) {
+        console.error("❌ Error fetching users:", error);
+        res.status(500).json({ message: "Internal server error: " + error.message }); // Include the error message
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)